### PR TITLE
1062 clinician sync

### PR DESCRIPTION
**Pull request by clemens-msupply**
Note, for the integration tests this needs the latest mSupply branch (test store needs to be a dispensary)

#1062

### CODE REVIEW

File: server/repository/migrations/postgres/2023-01-25T10-00_create_clinician_table/up.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    initials TEXT NOT NULL,
+    first_name TEXT,
+    registration_code TEXT,
+    category TEXT,
+    address1 TEXT,
+    address2 TEXT,
+    phone TEXT,
+    mobile TEXT,
+    email TEXT,
+    female BOOLEAN NOT NULL,
```
Comment (clemens-msupply) [inline server/repository/migrations/postgres/2023-01-25T10-00_create_clinician_table/up.sql:16]: Just noticed, this should be is_female, right? (its just `female` in mSupply)

Comment (roxy-dao) [reply server/repository/migrations/postgres/2023-01-25T10-00_create_clinician_table/up.sql:16]: `is_female` on om is more consistent

File: server/repository/migrations/postgres/2023-01-25T10-00_create_clinician_table/up.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    active BOOLEAN NOT NULL
```
Comment (roxy-dao) [inline server/repository/migrations/postgres/2023-01-25T10-00_create_clinician_table/up.sql:17]: same as above `is_active`

**Review by roxy-dao — APPROVED** [review]
Works great~

Thanks Clemens 😸

File: server/service/src/sync/translations/clinician.rs [hunk 3]
```diff
@@ -91,8 +91,8 @@ impl SyncTranslation for ClinicianTranslation {
             phone,
             mobile,
             email,
-            female,
-            active,
+            is_female: female,
```
Comment (roxy-dao) [inline server/service/src/sync/translations/clinician.rs:94]: #[serde(rename = "")

Comment (clemens-msupply) [reply server/service/src/sync/translations/clinician.rs]: good point, thanks!

**Pull request merged**